docs/src/main/sphinx/installation/cli.rst

```diff
@@ -13,28 +13,107 @@ Requirements
 The CLI requires a Java virtual machine available on the path.
 It can be used with Java version 8 and higher.
 
+The CLI uses the HTTP protocol and the :doc:`Trino client REST API
+</develop/client-protocol>` to communicate with the coordinator on the Trino
+cluster.
+
 Installation
 ------------
 
-Download :maven_download:`cli`, rename it to ``trino``,
-make it executable with ``chmod +x``, then run it:
+Download :maven_download:`cli`, rename it to ``trino``, make it executable with
+``chmod +x``, and run it to show the version of the CLI:
 
 .. code-block:: text
 
-    ./trino --server localhost:8080 --catalog hive --schema default
+    ./trino --version
+
+Run the CLI with ``--help`` or ``-h`` to see all available options.
+
+Running the CLI
+---------------
+
+The minimal command to start the CLI in interactive mode specifies the URL of
+the coordinator in the Trino cluster:
+
+.. code-block:: text
+
+    ./trino --server http://trino.example.com:8080
+
+Many other options are available to further configure how to start the CLI:
 
-Run the CLI with the ``--help`` option to see the available options.
+.. list-table::
+  :widths: 40, 60
+  :header-rows: 1
```

Comment:
Remove this table.

Reply:
this table assembles all the options that are useful for starting the CLI that are generic .. we want to build up the section above with more details and examples for it all and therefore need this table. We also want to expand the content in the table.

And then in addition we want to add a section about batch mode with examples and a table for all the related parameters for that .. we just want to confirm that this is okay before we build all that content

Reply:
Adding details about configs which isn't given by --help is okay. That is not what this table is though right now. It's just a single large table with mostly non-essential options (all of which can already be seen by running trino --help).

Reply:
Yeah .. we want to build this out more .. I will work with @Jessie212